backend/vehicles/src/common/helper/validateHash.helper.ts

```diff
@@ -0,0 +1,12 @@
+import * as CryptoJS from 'crypto-js';
+
+export const validateHash = (
+  queryString: string,
+  hashvalue: string,
+): boolean => {
+  const payBCHash: string = CryptoJS.MD5(
+    `${queryString}${process.env.PAYBC_API_KEY}`,
+  ).toString();
+  if (hashvalue === payBCHash) return true;
```

Comment:
should be able to restructure this as `return hashvalue === payBCHash`

Reply:
done